spec/controllers/insured/fdsh_ridp_verifications_controller_spec.rb

```diff
@@ -76,18 +75,83 @@
         expect(response).to have_http_status(:success)
         expect(response).to render_template("failed_validation")
       end
+    end
+  end
+end
+
+describe Insured::FdshRidpVerificationsController, "given an unauthorized user" do
+  let(:mock_user) do
+    instance_double(
+      User,
+      :has_hbx_staff_role? => false,
+      :person => mock_person
+    )
+  end
+  let(:mock_person) do
+    double(
+      policy_class: PersonPolicy,
+      :agent? => false
+    )
+  end
+  let(:mock_policy) do
```

Comment:
same comment as the others, this is not testing the new code

Reply:
Same comment as for the other controller.  However, I *am* in need of a set of negative cases.